integrations/operator/hack/fixture-operator-bot.yaml

```diff
@@ -0,0 +1,28 @@
+kind: role
```

Comment:
the irony of fate. the operator role is not supported by operator itself (v7 😅 )

Reply:
:see_no_evil: